lib/TPP/ConvertLinalgToTpp.cpp

```diff
@@ -35,31 +35,55 @@ struct ConvertGenericOpToTpp : public OpRewritePattern<linalg::GenericOp> {
   LogicalResult rewriteToTppOp(linalg::GenericOp linalgOp,
                                PatternRewriter &rewriter) const {
     SmallVector<Value> operands;
+    if (tpp::utils::isTppZero(linalgOp, &operands)) {
+      assert((operands.size() != 0 && operands.size() <= 2) &&
```

Comment:
Nit: `size > 0 && size <= 2` looks easier to understand.

I had to pause and think if `size` was signed or unsigned to see if there could be a problem.